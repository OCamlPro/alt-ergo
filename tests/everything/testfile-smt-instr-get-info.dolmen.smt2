(set-logic ALL)

(declare-const x Int)
(declare-const y Int)

(assert (= (* x x) 4))

(get-info :reason-unknown)
(check-sat)
(get-info :all-statistics)
(get-info :assertion-stack-levels)
(get-info :authors)
(get-info :error-behavior)
(get-info :name)
(get-info :reason-unknown)
(get-info :version)
(get-info :foo)