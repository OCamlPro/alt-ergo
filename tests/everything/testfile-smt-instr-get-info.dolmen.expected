(error "Invalid (get-info :reason-unknown)")

unknown
unsupported

unsupported

(:authors "Alt-Ergo developers")
(:error-behavior immediate-exit)
(:name "Alt-Ergo")
(:reason-unknown Incomplete)
(:version dev)
unsupported
